src/components/BreadCrumbs/BreadCrumbs.tsx

```diff
@@ -0,0 +1,46 @@
+import { FC } from 'react';
+import { Link, useLocation } from 'react-router-dom';
+
+import { Paths } from '@/enums';
+import { getCapitalizedWord } from '@/utils';
+
+import styles from './styles.module.scss';
+import homeIcon from '../../assets/images/icons/Home.svg';
```

Comment:
Doesn't it work with alias?
```suggestion
import homeIcon from '@/assets/images/icons/Home.svg';
```